app/scripts/modules/openstack/serverGroup/configure/serverGroup.configure.openstack.module.js

```diff
@@ -0,0 +1,15 @@
+'use strict';
+
+let angular = require('angular');
+
+module.exports = angular.module('spinnaker.openstack.serverGroup.configure', [
+  require('../../../core/account/account.module.js'),
+  require('./wizard/deployInitializer.controller.js'),
+  require('./ServerGroupCommandBuilder.js'),
+  require('./serverGroupConfiguration.service.js'),
+  require('./wizard/ServerGroupBasicSettings.controller.js'),
+  require('./wizard/Capacity.controller.js'),
+  require('./wizard/Clone.controller.js'),
+  require('./wizard/LoadBalancers.controller.js'),
+  require('./wizard/templateSelection.controller.js')
```

Comment:
Should the capitalization of the files be consistent e.g. `templateSelection.controller.js` and `Clone.controller.js`